client/lib/analytics/ad-tracking.js

```diff
@@ -1241,6 +1241,38 @@ function isSupportedCurrency( currency ) {
 	return Object.keys( EXCHANGE_RATES ).indexOf( currency ) !== -1;
 }
 
+/**
+ * Initializes the Facebook pixel.
+ *
+ * When the user is logged in, additional hashed data is being forwarded.
+ */
+function initFacebook() {
+	if ( user.get() ) {
+		initFacebookAdvancedMatching();
+	} else {
+		window.fbq( 'init', TRACKING_IDS.facebookInit ); // simple data set, no advanced matching
+	}
+}
+
+/**
+ * See https://developers.facebook.com/docs/facebook-pixel/pixel-with-ads/conversion-tracking#advanced_match
+ */
+function initFacebookAdvancedMatching() {
+	// All data must be in lowercase. Remove all spaces.
+	const fbRequirementsFormatter = function( str ) {
+		return str.toLowerCase().replace( / /g, '' );
+	};
+
+	const currentUser = user.get();
+	const advancedMatching = {
+		em: fbRequirementsFormatter( currentUser.email ),
```

Comment:
Do we want to hash the email? If so, you'll want the [`hashPii()` function](https://github.com/Automattic/wp-calypso/pull/24543) here. When I tested, the email is currently plain text.

Reply:
According to their documentation, this won't be necessary because they are hashing it via the pixel itself. I also looked to see if it makes a network call with the plain e-mail and I didn't see one.

Quote from them:

> Any customer data you pass through the Facebook JavaScript pixel is hashed locally on the browser before it goes to the Facebook servers for matching. If you use the image pixel, you are responsible for hashing the data prior to sending it to Facebook. This hashing process turns your data into short encrypted messages that are difficult to reverse engineer. We use these hashes to match people who have taken actions on your website with people on Facebook. Facebook typically deletes all matched and unmatched hashes within 48 hours after the matching process ends, unless we need to retain any data for troubleshooting or product improvement (  https://developers.facebook.com/docs/facebook-pixel/pixel-with-ads/conversion-tracking#advanced_match )

Did you see the e-mail in clear-text for debugging, or did you see it sent as plain text?